ckanext/dia/templates/uris/edit.html

```diff
@@ -54,7 +43,18 @@ <h2 class="module-heading"><i class="fa fa-info-circle"></i> {{ _('URIs for use
                 classes=["control-medium"],
                 attrs={"autocomplete": "off"})
             }}
-            <button class="btn btn-primary" type="submit">{{_('Submit')}}</button>
+
+            {{ form.info(_('Only tick this box if the entity that this URI represents is changing its remit or purpose.'), classes=['info-help-tight', 'small', 'bold']) }}
+            {{ form.checkbox('regenerate',
+                label=_("Regenerate URI?"),
+                checked=data.regenerate,
+                classes=[""])
+            }}
+
         </fieldset>
+        <div class="form-actions">
+            <a class="btn" href="/uri/">{{_('Cancel')}}</a>
```

Comment:
Kervin requested a change to the info text, moving the regen checkbox lower, and a cancel link